client/executor/wasmtime/src/tests.rs

```diff
@@ -226,8 +226,8 @@ fn deep_call_stack_wat(depth: usize) -> String {
 
 // We need two limits here since depending on whether the code is compiled in debug
 // or in release mode the maximum call depth is slightly different.
-const CALL_DEPTH_LOWER_LIMIT: usize = 65478;
-const CALL_DEPTH_UPPER_LIMIT: usize = 65514;
+const CALL_DEPTH_LOWER_LIMIT: usize = 65456;
+const CALL_DEPTH_UPPER_LIMIT: usize = 65509;
```

Comment:
So does that mean that we are actually using more stack size per level? And thus need reduce the limit?

Reply:
Yes, slightly more stack space is being used.

Should be fine though as it doesn't look multiplicative (I'm guessing `wasmtime` just has a few more local variables on the code path when entering into WASM or something like that.)